apps/judicial-system/web/src/components/ServiceAnnouncement/ServiceAnnouncements.tsx

```
@@ -158,4 +159,61 @@ const ServiceAnnouncement: FC<ServiceAnnouncementProps> = (props) => {
+            defendantName={defendant.name}
+          />
+        )}
+        {defendant.subpoenas?.map((subpoena) => (
+          <ServiceAnnouncement
+            key={`${subpoena.id}-${subpoena.created}`}
+            subpoena={subpoena}
+            defendantName={defendant.name}
+          />
+        ))}
+      </>
+    )
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)